common/domain/handler.go

```diff
@@ -345,6 +345,13 @@ func (d *handlerImpl) DescribeDomain(
 		IsGlobalDomain:  resp.IsGlobalDomain,
 		FailoverVersion: resp.FailoverVersion,
```

Comment:
Is this field marked as deprecated? Seems we're storing this info in FailoverInfo struct.